build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -146,7 +146,8 @@
       <AdditionalCoreSetupChannel>preview</AdditionalCoreSetupChannel>
       <AdditionalSharedFrameworkVersion>1.0.5</AdditionalSharedFrameworkVersion>
       <AdditionalSharedHostVersion>1.0.1</AdditionalSharedHostVersion>
-      <AdditionalHostFxrVersion>1.0.1</AdditionalHostFxrVersion>
+      <AdditionalHostFxrVersion Condition=" '$(OS)' != 'Windows_NT' ">1.0.1</AdditionalHostFxrVersion>
```

Comment:
Not sure about the semantics of this - Prepare.targets seems to use OSName instead of OS, do we want to be consistent?